deploy.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+UPDATE=$1
+DEST_DIR=$2
+FED_KEY=""
+MAIN_BLOCK_NUMBER=""
+SIDE_BLOCK_NUMBER=""
+
+cd $DEST_DIR/federator
+
+quit() {
+    if [ $1 -eq 1 ]; then
+        echo "ERROR: "$2
+    else
+        echo $2
+    fi
+    exit $1
+}
+
+config() {
+    cp $DEST_DIR/federator/config/config.sample.js $DEST_DIR/federator/config/config.js &&
+        sed -i "s|rsktestnet-kovan|rskmainnet|g" $DEST_DIR/federator/config/config.js &&
+        sed -i "s|kovan|ethmainnet|g" $DEST_DIR/federator/config/config.js &&
+        return 0
+    quit 1 "There was a problem setting the configuration, please verify."
+}
+
+dependencies() {
+    echo "Installing required packages ..."
+    npm install &&
+        echo "Packages installed" &&
+        return 0
+    quit 1 "There was an error installing the needed packages, please check"
+}
+
+key() {
```

Comment:
It'd be great if the user was provided with a way to share the private key with the script in a non-interactive way.

Reply:
I changed the method

Reply:
This is better.